experiments/music/lib/psyparticipant.py

```diff
@@ -0,0 +1,24 @@
+"""
+Display the dialogue for the psychopy participant ID
+This is seperated from the `psy.py` to avoid a fullscreen
+window covering the dialogue box.
```

Comment:
Will that really happen? I wouldn't put it beside PsychoPy but at least usually, you can just ask it to wait, right?

Reply:
yep, that happened to me, at least last year, i think it depends on the video backend (i was using glfw, so the prob might not exist in pygame, but i'll try whatever causes the least amount of problems and is fastest)